File: src/easy/compare_string_length.zig
const expectEqual = @import("std").testing.expectEqual;

pub fn compareStringLength(a: []const u8, b: []const u8) bool {
	return a.len == b.len;
}
test "Do strings have same length" {
try expectEqual(compareStringLength("AB", "CD"), true);
try expectEqual(compareStringLength("ABC", "DE"), false);
try expectEqual(compareStringLength("hello", "edabit"), false);
try expectEqual(compareStringLength("meow", "woof"), true);
try expectEqual(compareStringLength("jrnvjrnnt", "cvjknfjvmfvnfjn"), false);
try expectEqual(compareStringLength("jkvnjrt", "krnf"), false);
try expectEqual(compareStringLength("Facebook", "Snapchat"), true);
}

File: src/easy/compare_string_length.rs
pub fn compare_string_length(a: &str, b: &str) -> bool {
    a.len() == b.len()
}
#[cfg(test)]
mod tests {
    use super::compare_string_length;
    #[test]
    fn test_compare_string_length() {
        assert_eq!(compare_string_length("AB", "CD"), true);
        assert_eq!(compare_string_length("ABC", "DE"), false);
        assert_eq!(compare_string_length("hello", "edabit"), false);
        assert_eq!(compare_string_length("meow", "woof"), true);
        assert_eq!(compare_string_length("jrnvjrnnt", "cvjknfjvmfvnfjn"), false);
        assert_eq!(compare_string_length("jkvnjrt", "krnf"), false);
        assert_eq!(compare_string_length("Facebook", "Snapchat"), true);
    }
}
